config/default.yml

```
@@ -362,6 +362,11 @@ Rails/IgnoredSkipActionFilterOption:
   Include:
     - app/controllers/**/*.rb
 
+Rails/ImplementedMigration:
+  Description: 'Checks for migrations that have a typo in a method name or are half implemented.'
+  Enabled: false
+  VersionAdded: '2.10'
```

Can you add `Include` path?

```suggestion
  VersionAdded: '2.10'
  Include:
    - db/migrate/*.rb
```